specs/core/1_custody-game.md

```diff
@@ -222,6 +225,22 @@ def get_custody_chunk_bit(key: BLSSignature, chunk: bytes) -> bool:
     return get_bitfield_bit(hash(challenge.responder_key + chunk), 0)
 ```
 
+### `get_chunk_bits_root`
+
+```python
+def get_chunk_bits_root(chunk_bits: Bitfield, chunk_count: int) -> bool:
+    bits_to_aggregate = 2**int(math.log2(next_power_of_two(chunk_count)) - 9)
+
+    aggregated_bits = b'\0' * 64
```

Comment:
Why 64? Should we add a constant here?
`b'\x00'` is more readable imo and is how we've defined 0's in bytestrings in phase0